data/linux-bridge/002-linux-bridge.yaml

```diff
@@ -17,6 +17,8 @@ spec:
         name: kube-cni-linux-bridge-plugin
         tier: node
         app: cni-plugins
+      annotations:
+        description: LinuxBridge manages bridge creation and configures them to the host and the container
```

Comment:
"configures them to the host and the container"?

Can we comment that it "installs 'bridge' CNI on cluster nodes, so it can be later used to attach Pods/VMs to Linux bridges"

Something like that

Reply:
I'm a terrible marketing person =)